include/cub3d.h

```diff
@@ -103,6 +103,18 @@
 #  define MMAP_BORDER_SIZE 5
 # endif
 
+# ifndef RAYS_NB
+#  define RAYS_NB 1000
```

Comment:
This could have been WIDTH instead.
So if someone changes the width later nothing is broken.